<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1383</width>
    <height>971</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #006d77;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #006d77;
color:#edf6f9;
font-size: 28px;
font-weight:500;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>icons_setup/icons/face-scan.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #006d77;
color:#edf6f9;
font-size: 28px;
font-weight:500;</string>
          </property>
          <property name="text">
           <string>Face Recognition Lab</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browse">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#edf6f9;
border-radius: 5px;
color:#006d77;
font-size: 16px;
font-weight:500;</string>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons_setup/icons/image.png</normaloff>icons_setup/icons/image.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="detection_output">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#edf6f9;
border-radius: 5px;
color:#006d77;
font-size: 16px;
font-weight:500;</string>
          </property>
          <property name="text">
           <string>Detect</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons_setup/icons/face-recognition.png</normaloff>icons_setup/icons/face-recognition.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="recognize_button">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #edf6f9;
    border-radius: 5px;
    color: #006d77;
    font-size: 16px;
    font-weight: 500;
}</string>
          </property>
          <property name="text">
           <string>Recognize</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons_setup/icons/face-id.png</normaloff>icons_setup/icons/face-id.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#edf6f9;
border-radius: 5px;
color:#006d77;
font-size: 16px;
font-weight:500;</string>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons_setup/icons/undo (1).png</normaloff>icons_setup/icons/undo (1).png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>23</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #006d77;
color:#edf6f9;
font-size: 18px;
font-weight:500;</string>
              </property>
              <property name="text">
               <string>Input:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="input_widget" native="true">
            <property name="minimumSize">
             <size>
              <width>650</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:#edf6f9;
border-radius: 5px;</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_29">
             <item>
              <layout class="QVBoxLayout" name="input_layout"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #006d77;
color:#edf6f9;
font-size: 18px;
font-weight:500;</string>
              </property>
              <property name="text">
               <string>Detection Output:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QWidget" name="detection_widget" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>700</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#edf6f9;
border-radius: 5px;</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_30">
               <item>
                <layout class="QVBoxLayout" name="detection_layout"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #006d77;
color:#edf6f9;
font-size: 18px;
font-weight:500;</string>
              </property>
              <property name="text">
               <string>Recognition Output:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="recognition_output" native="true">
            <property name="styleSheet">
             <string notr="true">background-color:#edf6f9;
border-radius: 5px;</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="recognition_layout"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="result_widget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #edf6f9;
border-radius: 10px;</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #006d77;
color:#edf6f9;
font-size: 20px;
font-weight:500;
border: None;</string>
             </property>
             <property name="text">
              <string>This Image </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="match_precentage">
             <property name="styleSheet">
              <string notr="true">background-color: #006d77;
color:#edf6f9;
font-size: 20px;
font-weight:500;
border: None;</string>
             </property>
             <property name="text">
              <string>100%</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true">background-color: #006d77;
color:#edf6f9;
font-size: 20px;
font-weight:500;
border: None;</string>
             </property>
             <property name="text">
              <string>Matches</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="match_name">
             <property name="styleSheet">
              <string notr="true">background-color: #006d77;
color:#edf6f9;
font-size: 20px;
font-weight:500;
border: None;</string>
             </property>
             <property name="text">
              <string>Ahmed</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
